worker-sys/Cargo.toml

```diff
@@ -13,7 +13,7 @@ wasm-bindgen.workspace = true
 cfg-if = "1.0.0"
 
 [dependencies.web-sys]
-version = "0.3.63"
+version = ">=0.3.70"
```

Comment:
Hopefully this prevents folks with an older version in their Cargo.lock from running into the reverse issue.